tests/conftest.py

```diff
@@ -0,0 +1,31 @@
+import os
+import sys
+
+import pytest
+from _pytest.monkeypatch import MonkeyPatch
+
+TESTS_DIR = os.path.abspath(os.path.dirname(__file__))
+PKG_DIR = os.path.dirname(TESTS_DIR)
+
+sys.path.insert(0, PKG_DIR)
+
+
+CACHE_KEY = 'pip-cache/is-cache-built'
+
+
+@pytest.fixture(scope='session')
+def pip_cache_dir(request):
+    path = request.config.cache.makedir('pip-cache-data')
```

Comment:
This'll create a directory under `.pytest_cache/d/`, which'll remain in between test runs. To clear the cache, `py.test --cache-clear` can be used; though, deleting `.pytest_cache` would prolly work just as well :P